composer.json

```diff
@@ -152,10 +152,13 @@
         "predis/predis": "Required to use the predis connector (^1.1.2).",
         "psr/http-message": "Required to allow Storage::put to accept a StreamInterface (^1.0).",
         "pusher/pusher-php-server": "Required to use the Pusher broadcast driver (^5.0|^6.0).",
+        "symfony/amazon-mailer": "Required to enable support for the SES mail transport (^6.0).",
         "symfony/cache": "Required to PSR-6 cache bridge (^6.0).",
         "symfony/filesystem": "Required to enable support for relative symbolic links (^6.0).",
-        "symfony/psr-http-message-bridge": "Required to use PSR-7 bridging features (^2.0).",
-        "wildbit/swiftmailer-postmark": "Required to use Postmark mail driver (^3.0)."
+        "symfony/http-client": "Required to enable support for the Symfony mail transports (^6.0).",
```

Comment:
So will people need this to use any mail features or just API based mail transports?

Reply:
You only need this for the API based transports.
Smtp Based transport should be working fine without it. (it is also only a dev-requirement on the repositories itself)

Reply:
Yes only API based transports. I added "API" to the sentence to clarify this.